src/quo/components/avatars/user_avatar/view.cljs

```diff
@@ -55,10 +57,10 @@
            status-indicator?   true
            online?             true
            ring?               true
-           customization-color :blue}
+           customization-color :blue
+           full-name           "Your Name"}
     :as   props}]
-  (let [full-name          (or full-name "Your Name")
```

Comment:
`:or` will not handle the case where `full-name` is passed as nil (e.g. used with `(when ...)`). I guess would be better to keep it as was, wdyt?